masterlist.yaml

```
@@ -808,3 +808,28 @@ plugins:
       - 'NOM3.0.esp'
       - 'NOM3_PT SO.esp'
       - 'NoM_Creatures Loot Standard.esp'
+
+  ### Other Stuff (sort later) ###
+  - name: 'Better Propylon Teleport Warp.ESP'
```

The locations & messages can be refactored with a regex plugin:

```suggestion
  - name: 'Better Propylon Teleport Warp(-Master Index)?\.esp'
```